CONTRIBUTING.md

```diff
@@ -196,12 +196,11 @@ Please follow all style guides to the best of your abilities.
 Pycytominer uses [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard for commit messages to aid in automatic changelog generation.
 We prepare commit messages that follow this standard using [commitizen](https://commitizen-tools.github.io/commitizen/), which comes with the poetry dev dependencies.
 
-### Python style guide
+### Python code quality guide
 
-For python code style, we use [black](https://github.com/psf/black).
-Please use black before committing any code.
-We will not accept code contributions that do not use black.
-Configuring your [dev environment](#dev-environments) as described above will ensure your code is formatted correctly automatically (using a tool called [pre-commit](https://pre-commit.com/)).
+For python code linting and formatting, we use [ruff](https://docs.astral.sh/ruff/) which is a Rust-based linter/formatter with drop-in parity with Black and Flake8.
```

Comment:
Ruff describes itself as "drop-in parity with black" in its README. But seems you are correct that there are some minor deviations. How about "near drop-in parity"?

Reply:
This sounds okay to me.

Aside, generally I understood there to be an indication from Ruff's documentation that it will likely deviate more from Black in the future (and possibly other tools it mirrors). Given that the documentation can't easily be checked/tested here it felt likely that this sentence could go untouched alongside those types of changes, potentially making inaccurate statements about these tools (i.e. what if a year from now Ruff is considered very different from Black or Flake8?).

I'm also sensitive to the potential of creating or reinforcing an [association fallacy](https://en.wikipedia.org/wiki/Association_fallacy) here for developers or users. Specifically, tools which implement the specification of another project aren't guaranteed the same operation or prior success. I state this mostly out of care for the span of control we have within Pycytominer.

Reply:
Good points.

Changed language to:
> Ruff includes a python code formatter similar to Black.

And:
> For python code linting, we also use [ruff](https://docs.astral.sh/ruff/), which can perform same linting checks as Flake8.